content/en/Getting started/Assets/_index.md

```diff
@@ -39,12 +39,9 @@ The asset screen prompts you for the following information:
 - Asset Image: Use it to help identify what you need from a list of assets.
 - Asset Title: Set up a descriptive name to attract attention from the best pentesters.
 - [Asset Type](./asset-type): Select one of the options described in the linked page.
-- Asset Scoping: Review the guidance on:
-  - [Asset Size](./asset-size)
-  - [Coverage and Scope](./coverage)
 - [Asset Description](./asset-description): Add information that can help your
   pentesters fully analyze your asset.
-- [Asset Documentation](./asset-description/#asset-documentation): Upload documentation,
+- [Attachment(s)](./asset-description/#attachments): Upload documentation,
```

Comment:
While I realize users might have only one attachment, the Google Developer Style Guide discourages parenthesis, https://developers.google.com/style/plurals-parentheses

```suggestion
- [Attachments](./asset-description/#attachments): Upload documentation,
```